spotbugs/src/main/java/edu/umd/cs/findbugs/detect/ConstructorThrow.java

```diff
@@ -79,49 +87,47 @@ public void visit(Method obj) {
      */
     @Override
     public void sawOpcode(int seen) {
-        if (isFinalClass || isFinalFinalizer || alreadyReported)
+        if (isFinalClass || isFinalFinalizer)
             return;
         if (isFirstPass) {
             tryCollectMethod(seen);
         } else {
             if (isConstructor() || methodCalledFromCtor()) {
                 if (seen == Const.ATHROW)
-                    reportCTBug();
+                    accumulateBug();
             }
         }
     }
 
     /** Collects all the methods that are called from the constructor */
     private void tryCollectMethod(int seen) {
-        if (isMethodCall(seen)) {
-            String classConstantOperand = "";
-            try {
-                classConstantOperand = getClassConstantOperand();
-            } catch (Exception e) {
-                bugReporter.logError("Seen OPcode and failed to get ClassConstantOperand: " + seen, e);
-            }
-            String method = classConstantOperand + "." + getNameConstantOperand() + " : " + getSigConstantOperand();
-            // Not interested in object superctor
-            if (!"java/lang/Object.<init> : ()V".equals(method)) {
-                calledFromCtor.add(method);
-            }
+        if (!isMethodCall(seen))
```

Comment:
`if` statements should always have braces, even for single lines, to avoid [goto fail](https://dwheeler.com/essays/apple-goto-fail.html) errors.

Reply:
Thanks, @ThrawnCA for pointing out these remarks, and for the reference too. (I am going to reference that goto fail in my thesis, so thanks again).